AzureLinux verticals are failing

```
    /__w/1/s/src/runtime/src/coreclr/pal/src/thread/process.cpp:2572:16: error: use of undeclared identifier 'MEMBARRIER_CMD_PRIVATE_EXPEDITED'
       2572 |         mask & MEMBARRIER_CMD_PRIVATE_EXPEDITED)
            |                ^
      [ 85%] Building CXX object pal/src/CMakeFiles/coreclrpal.dir/arch/amd64/signalhandlerhelper.cpp.o
      /__w/1/s/src/runtime/src/coreclr/pal/src/thread/process.cpp:2575:24: error: use of undeclared identifier 'MEMBARRIER_CMD_REGISTER_PRIVATE_EXPEDITED'
       2575 |         if (membarrier(MEMBARRIER_CMD_REGISTER_PRIVATE_EXPEDITED, 0, 0) == 0)
            |                        ^
      [ 85%] Built target eventing_headers
      /__w/1/s/src/runtime/src/coreclr/pal/src/thread/process.cpp:2640:33: error: use of undeclared identifier 'MEMBARRIER_CMD_PRIVATE_EXPEDITED'
       2640 |         int status = membarrier(MEMBARRIER_CMD_PRIVATE_EXPEDITED, 0, 0);
            |                                 ^
      3 errors generated.
      make[3]: *** [pal/src/CMakeFiles/coreclrpal.dir/build.make:944: pal/src/CMakeFiles/coreclrpal.dir/thread/process.cpp.o] Error 1
      make[3]: *** Waiting for unfinished jobs....

```

Maybe regressed with https://github.com/dotnet/sdk/commit/affada6e1e99189c71777602c6764b1e8d61b8b1?

--- a/eng/pipelines/templates/stages/vmr-build.yml
+++ b/eng/pipelines/templates/stages/vmr-build.yml
@@ -733,8 +733,8 @@ stages:
             sign: ${{ variables.signEnabled }}
             pool: ${{ parameters.pool_Linux }}
             container:
-              name: ${{ variables.azurelinuxX64AlpineCrossContainerName }}
-              image: ${{ variables.azurelinuxX64AlpineCrossContainerImage }}
+              name: ${{ variables.azurelinuxX64MuslCrossContainerName }}
+              image: ${{ variables.azurelinuxX64MuslCrossContainerImage }}
             crossRootFs: '/crossrootfs/x64'
             targetOS: linux-musl
             targetArchitecture: x64
@@ -748,8 +748,8 @@ stages:
             sign: ${{ variables.signEnabled }}
             pool: ${{ parameters.pool_Linux }}
             container:
-              name: ${{ variables.azurelinuxArmAlpineCrossContainerName }}
-              image: ${{ variables.azurelinuxArmAlpineCrossContainerImage }}
+              name: ${{ variables.azurelinuxArmMuslCrossContainerName }}
+              image: ${{ variables.azurelinuxArmMuslCrossContainerImage }}
             crossRootFs: '/crossrootfs/arm'
             targetOS: linux-musl
             targetArchitecture: arm
@@ -763,8 +763,8 @@ stages:
             sign: ${{ variables.signEnabled }}
             pool: ${{ parameters.pool_Linux }}
             container:
-              name: ${{ variables.azurelinuxArm64AlpineCrossContainerName }}
-              image: ${{ variables.azurelinuxArm64AlpineCrossContainerImage }}
+              name: ${{ variables.azurelinuxArm64MuslCrossContainerName }}
+              image: ${{ variables.azurelinuxArm64MuslCrossContainerImage }}
             crossRootFs: '/crossrootfs/arm64'
             targetOS: linux-musl
             targetArchitecture: arm64

--- a/eng/pipelines/templates/variables/vmr-build.yml
+++ b/eng/pipelines/templates/variables/vmr-build.yml
@@ -127,7 +127,7 @@ variables:
 - name: linuxBionicCrossContainerName
   value: linuxBionicCrossContainer
 - name: linuxBionicCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-android-openssl
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-android-openssl-amd64
 
 - name: browserCrossContainerName
   value: browserCrossContainer

--- a/eng/pipelines/templates/variables/vmr-build.yml
+++ b/eng/pipelines/templates/variables/vmr-build.yml
@@ -104,19 +104,19 @@ variables:
 - name: azurelinuxArm64CrossContainerImage
   value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-arm64
 
-- name: azurelinuxX64AlpineCrossContainerName
-  value: azurelinuxX64AlpineCrossContainer
-- name: azurelinuxX64AlpineCrossContainerImage
+- name: azurelinuxX64MuslCrossContainerName
+  value: azurelinuxX64MuslCrossContainer
+- name: azurelinuxX64MuslCrossContainerImage
   value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-amd64-musl
 
-- name: azurelinuxArmAlpineCrossContainerName
-  value: azurelinuxArmAlpineCrossContainer
-- name: azurelinuxArmAlpineCrossContainerImage
+- name: azurelinuxArmMuslCrossContainerName
+  value: azurelinuxArmMuslCrossContainer
+- name: azurelinuxArmMuslCrossContainerImage
   value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-arm-musl
 
-- name: azurelinuxArm64AlpineCrossContainerName
-  value: azurelinuxArm64AlpineCrossContainer
-- name: azurelinuxArm64AlpineCrossContainerImage
+- name: azurelinuxArm64MuslCrossContainerName
+  value: azurelinuxArm64MuslCrossContainer
+- name: azurelinuxArm64MuslCrossContainerImage
   value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-arm64-musl
 
 - name: androidCrossContainerName

--- a/eng/pipelines/templates/variables/vmr-build.yml
+++ b/eng/pipelines/templates/variables/vmr-build.yml
@@ -127,7 +127,7 @@ variables:
 - name: linuxBionicCrossContainerName
   value: linuxBionicCrossContainer
 - name: linuxBionicCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-android-openssl-amd64
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-android-openssl
 
 - name: browserCrossContainerName
   value: browserCrossContainer

--- a/eng/pipelines/templates/variables/vmr-build.yml
+++ b/eng/pipelines/templates/variables/vmr-build.yml
@@ -92,52 +92,52 @@ variables:
 - name: azurelinuxX64CrossContainerName
   value: azurelinuxX64CrossContainer
 - name: azurelinuxX64CrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-cross-amd64
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-amd64
 
 - name: azurelinuxArmCrossContainerName
   value: azurelinuxArmCrossContainer
 - name: azurelinuxArmCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-cross-arm
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-arm
 
 - name: azurelinuxArm64CrossContainerName
   value: azurelinuxArm64CrossContainer
 - name: azurelinuxArm64CrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-cross-arm64
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-arm64
 
 - name: azurelinuxX64AlpineCrossContainerName
   value: azurelinuxX64AlpineCrossContainer
 - name: azurelinuxX64AlpineCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-cross-amd64-alpine
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-amd64-musl
 
 - name: azurelinuxArmAlpineCrossContainerName
   value: azurelinuxArmAlpineCrossContainer
 - name: azurelinuxArmAlpineCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-cross-arm-alpine
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-arm-musl
 
 - name: azurelinuxArm64AlpineCrossContainerName
   value: azurelinuxArm64AlpineCrossContainer
 - name: azurelinuxArm64AlpineCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-cross-arm64-alpine
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-arm64-musl
 
 - name: androidCrossContainerName
   value: androidCrossContainer
 - name: androidCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-cross-android-amd64
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-android-amd64
 
 - name: linuxBionicCrossContainerName
   value: linuxBionicCrossContainer
 - name: linuxBionicCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-android-openssl
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-android-openssl-amd64
 
 - name: browserCrossContainerName
   value: browserCrossContainer
 - name: browserCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-webassembly-amd64
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-webassembly-amd64
 
 - name: wasiCrossContainerName
   value: wasiCrossContainer
 - name: wasiCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net9.0-webassembly-amd64
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-webassembly-amd64
 
 - name: almaLinuxName
   value: AlmaLinux8

--- a/eng/pipelines/templates/variables/vmr-build.yml
+++ b/eng/pipelines/templates/variables/vmr-build.yml
@@ -127,7 +127,7 @@ variables:
 - name: linuxBionicCrossContainerName
   value: linuxBionicCrossContainer
 - name: linuxBionicCrossContainerImage
-  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-android-openssl-amd64
+  value: mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-net10.0-cross-android-openssl-amd64
 
 - name: browserCrossContainerName
   value: browserCrossContainer

